compiler/resolution/functionResolution.cpp

```diff
@@ -5459,7 +5459,8 @@ static void resolveMoveForRhsSymExpr(CallExpr* call) {
       // todo: not all const if it is zippered and one of iterators is var
       if (isReferenceType(rhsType)                == false &&
           isTupleContainingAnyReferences(rhsType) == false &&
-          rhsType->symbol->hasFlag(FLAG_ARRAY)    == false) {
+          rhsType->symbol->hasFlag(FLAG_ARRAY)    == false &&
+          rhsType->symbol->hasFlag(FLAG_COPY_MUTATES) == false) {
```

Comment:
I would prefer to get rid of "== false" in the above, while we are here.

Reply:
I'm going to leave it for now but we should at some point have a broader discussion about code style. While I think if (something == true) is unnecessarily verbose, if (something == false) seems about as reasonable to me as writing if (!something).